dev/com.ibm.ws.concurrent.persistent_fat_multiple/build.gradle

```diff
@@ -8,6 +8,71 @@
  * Contributors:
  *     IBM Corporation - initial API and implementation
  *******************************************************************************/
+ 
+def isLocalBuild = Boolean.valueOf(System.getProperty("fat.test.localrun"))
+def dbType = System.properties.getProperty("fat.bucket.db.type")
 
+configurations {
+  anonymousJDBC
+}
 
-addRequiredLibraries.dependsOn addDerby
+dependencies {
+  //Test container libraries
+  requiredLibs 'org.testcontainers:testcontainers:1.12.0',
+               'org.testcontainers:database-commons:1.12.0',
+               'org.testcontainers:jdbc:1.12.0',
+               'org.testcontainers:oracle-xe:1.12.0',
+               'org.testcontainers:mssqlserver:1.12.0',
+               'org.testcontainers:postgresql:1.12.0',
+               'org.testcontainers:mockserver:1.12.0',
+               'org.testcontainers:db2:1.12.0',
+               'org.apache.commons:commons-compress:1.18',
+               'org.rnorth.duct-tape:duct-tape:1.0.7',
+               'org.rnorth.visible-assertions:visible-assertions:2.1.2',
+               'org.rnorth:tcp-unix-socket-proxy:1.0.2',
+               'net.java.dev.jna:jna:5.2.0',
+               'org.slf4j:slf4j-api:1.7.7',
+               'org.slf4j:slf4j-jdk14:1.7.7'
+
+  //JDBC driver - depends on database type
+  switch(dbType){
+    case 'DB2':
+      anonymousJDBC 'com.ibm.db2:jcc:11.1.4.4'
+      break
+    case 'Postgre':
+      anonymousJDBC 'org.postgresql:postgresql:42.2.5'
+      break
+    case 'Oracle':
+      //If we are doing a personal build then prereq.dbtest will be available to get oracle jdbc driver
+      if (!isLocalBuild){
+        System.properties.setProperty("fat.oracle.is.available", "true")
+        anonymousJDBC files("../prereq.dbtest/lib/Oracle/jars/Oracle/18.3.0.0.0/ojdbc8_g.jar")
+      
+      //Otherwise, let user know they cannot do a local build using oracle test container
+      //and set fat.oracle.is.available = false so tests are skipped.
+      } else {
+      	System.out.println("Notice: Using an Oracle test container while running locally is not currently supported.")
+        System.properties.setProperty("fat.oracle.is.available", "false")
+        
+        //TODO Use the following to test locally if you have access to WS-CD-Open
+        //anonymousJDBC files("../../../WS-CD-Open/dev/prereq.dbtest/lib/Oracle/jars/Oracle/18.3.0.0.0/ojdbc8_g.jar")
+      }
+      
+      break
+    case 'SQLServer':
+      anonymousJDBC 'com.microsoft.sqlserver:mssql-jdbc:7.2.2.jre8'
+      break
+  }
+}
+
+task addAnonymousDriver(type: Copy) {
+  shouldRunAfter jar
+  from configurations.anonymousJDBC
+  into new File(autoFvtDir, 'publish/shared/resources/jdbc')
+  rename '.*.jar', 'driver.jar'
```

Comment:
lets copy all of the JDBC drivers into the jdbc folder, but instead of naming them to `driver.jar` we can just strip off the versions so we have things like `db2jcc.jar` and `sqlserver.jar` (or whatever the base names are)